cartography/intel/github/repos.py

```diff
@@ -179,10 +180,10 @@ def _transform_repo_languages(repo_url, repo, repo_languages):
 
 
 @timeit
-def load_github_repos(session, update_tag, repo_data):
+def load_github_repos(neo4j_session, update_tag, repo_data):
```

Comment:
While I'm here I figured I'd standardize on clarifying what type of session object this is, like we do in the other syncs.